src/configure.ts

```diff
@@ -1,12 +1,12 @@
-import { graphqlExpress, graphiqlExpress } from 'apollo-server-express';
+import { Request } from 'express';
 import { createContext } from './context';
 import { GatewayOptions, GraphQLServer, Service } from './types';
 import { getSubscriptionEndpoint, tryParseJson, defaultErrorLogger } from './utils';
 import { createSubscription } from './subscription';
 import { createSchema } from './schema';
 import { defaultGraphiQLPath, defaultRootPath, defaultApiCreator } from './constants';
 
-const { apolloUploadExpress } = require('apollo-upload-server');
+const { ApolloServer } = require('apollo-server-express');

```

Comment:
Any reason for not using a standard `import` statement here? For `apollo-upload-server` the `require` was only chosen due to missing type decl iirc.

Reply:
same reason here. the issue is described here https://github.com/apollographql/apollo-server/issues/927#issuecomment-445537360
> Adding "skipLibCheck": true, has helped as a workaround, but the issue is still topical.

Reply:
Hm that does not sound right - have you tried removing `node_modules` and `package-lock.json` and reinstalling? I tried before to use the `apollo-server-express` v2 and it just worked with standard `import`.

Reply:
this didn't help
```sh
> tsc --project tsconfig.json

node_modules/apollo-server-core/dist/ApolloServer.d.ts:5:8 - error TS1259: Module '"/home/dmytroborysovskyi/WebstormProjects/gqlx-apollo-express-server/node_modules/apollo-server-core/node_modules/@types/ws/index"' can only be default-imported using the 'esModuleInterop' flag

5 import WebSocket from 'ws';
         ~~~~~~~~~

  node_modules/apollo-server-core/node_modules/@types/ws/index.d.ts:265:1
    265 export = WebSocket;
        ~~~~~~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

node_modules/apollo-server-express/dist/ApolloServer.d.ts:1:8 - error TS1259: Module '"/home/dmytroborysovskyi/WebstormProjects/gqlx-apollo-express-server/node_modules/@types/express/index"' can only be default-imported using the 'esModuleInterop' flag

1 import express from 'express';
         ~~~~~~~

  node_modules/@types/express/index.d.ts:115:1
    115 export = e;
        ~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

node_modules/apollo-server-express/dist/ApolloServer.d.ts:2:8 - error TS1259: Module '"/home/dmytroborysovskyi/WebstormProjects/gqlx-apollo-express-server/node_modules/@types/cors/index"' can only be default-imported using the 'esModuleInterop' flag

2 import corsMiddleware from 'cors';
         ~~~~~~~~~~~~~~

  node_modules/@types/cors/index.d.ts:34:1
    34 export = e;
       ~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.


Found 3 errors.
```

Reply:
So did you try the `skipLibCheck`? Because we don't care if the libs are properly typed - as long as they are typed.

Reply:
yes, works fine with `"skipLibCheck": true`

Reply:
no more issues with `skipLibCheck`